docs/tpm.md

```diff
@@ -78,6 +93,7 @@ Curl version 7.86
 	```
 10. Run the command to update symlinks and rebuild the library cache:
 	```
+	grep -qxF '/usr/local/lib/' /etc/ld.so.conf.d/libc.conf || echo /usr/local/lib/ | sudo tee -a /etc/ld.so.conf.d/libc.conf
```

Comment:
This step still required even if we don't require both openssl version co-exist ? If yes, update in cse.md as well.